src/aerospike_clj/client.clj

```diff
@@ -291,6 +298,27 @@
                       (:transcoder conf identity))]
       (register-events d db "read-batch" nil start-time)))))
 
+(defn exists-batch
+  "Check for each key in a batch if it exists. `indices` is a collection of maps
+  of the form `{:index \"key-name\" :set \"set-name\"}`. The result is a
+  vector of booleans in the same order as indices."
+  ([db indices]
+   (exists-batch db indices {}))
+  ([db indices conf]
+   (let [client (get-client db)
+         op-future (d/deferred)
+         start-time (System/nanoTime)
+         indices (utils/v->array Key (mapv #(create-key (:index %) (:dbns db) (:set %)) indices))]
+     (.exists ^AerospikeClient client
+              ^EventLoop (.next ^NioEventLoops (:el db))
+              (reify-exists-array-listener op-future)
+              ^BatchPolicy (:policy conf)
+              ^"[Lcom.aerospike.client.Key;" indices)
+     (let [d (d/chain' op-future
+                       #(vec %)
+                       (:transcoder conf identity))]
+       (register-events d db "read-batch" nil start-time)))))
```

Comment:
do we want a "exists-batch" metric?

Reply:
I personally don't think we need this granularity, but your call.